exports_internal_test.go

```
@@ -0,0 +1,15 @@
+package dmorph
+
+import "database/sql"
+
+// The exported names in this file are only used for internal testing and are not part of the public API.
+
+var (
+	TapplyStepsStream    = applyStepsStream
```

golangci-lint (2.2.2)

TapplyStepsStream is a global variable

(gochecknoglobals)